src/components/RevokeKeyDialog/RevokeKeyDialog.tsx

```diff
@@ -34,15 +34,19 @@ export const RevokeKeyDialog = ({ row }: RevokeKeyDialogProps) => {
       // At this point, spvWalletClient is defined; using non-null assertion.
       return await spvWalletClient!.RevokeAccessKey(row.original.id);
     },
-    onSuccess: () =>
+    onSuccess: () => {
       queryClient.invalidateQueries({
         queryKey: ['accessKeys'],
-      }),
+      });
+      toast.success('Access key revoked');
+    },
+    onError: () => {
+      toast.error('Error revoking access key');
+    },
```

Comment:
These changes are not related to this task, are they?

Reply:
It's the same bug as for destination view. I can create a separate task for this.

Reply:
No - don't do that, but instead, please describe in the PR description what actually has been done here and how to test it.

Reply:
Added description